docker/build.mk

```diff
@@ -68,6 +68,15 @@ dispatcher-image: base-image
     $(call cache_from,${BASE_TAG}/dispatcher-image) \
     docker/dispatcher-image
 
+measure-worker: base-runner
+	docker build \
+    --tag $(BASE_TAG)/measure-worker \
```

Comment:
super nit: docker build lines is not aligned

Reply:
I think I'm missing something because I can't tell what isn't aligned :-)
The first line uses a tab and each subsequent line uses four spaces, this is what we do in other targets also.
Could you maybe edit it using the online interface or specify how it should be alighned.?

Reply:
never mind then, looks like on the github code review view a tab is 6 spaces... :)